guide_to_writing_techreports.tex

```diff
@@ -253,19 +253,19 @@ \subsection{Chemistry}
 website (\url{http://www.iupac.org}).  Where unusual compounds are being used, it is often advisable to list their CAS numbers or INChI codes to make it easier for others to find compound information.
 
 The correct symbols should be used for chemical reactions,
-particularly in reversible reactions like
-\begin{displaymath}
-  \ce{A + 2B <=> AB2}
-\end{displaymath}
+particularly in reversible reactions like Reaction~\ref{eq:rxn}
```

Comment:
We introduce some ambiguity here - do we use different numbers for reactions and equations? I've always felt that a reaction _was_ an equation. I think this needs some talking through.

Reply:
I've tried to find examples in textbooks and papers, but to no avail. I suggest perhaps if the reaction isn't referenced again, don't give it a number. If it is referenced again, give it a number continuing from the equations. If many equations and reactions are mentioned in the text, perhaps use roman numerals for the latter?